projects/browser-extension/vue-app/src/views/apple-ios.view.vue

```diff
@@ -17,13 +17,16 @@
         No touch icons detected.
       </p>
       <properties-list>
-        <template v-for="icon in touchIcons">
-          <dt :key="`${icon.url}-key`">
```

Comment:
I did this, moving the `key` to the `<template>`, already for all views in the web-app 👍